src/component/environments/EnvironmentRow/EnvironmentRow.tsx

```diff
@@ -0,0 +1,26 @@
+import { useDragItem, MoveListItem } from 'hooks/useDragItem';
+import { Row } from 'react-table';
+import { TableRow } from '@mui/material';
+import { TableCell } from 'component/common/Table';
+import { useSearchHighlightContext } from 'component/common/Table/SearchHighlightContext/SearchHighlightContext';
+
+interface IEnvironmentRowProps {
+    row: Row;
+    moveListItem: MoveListItem;
+}
+
+export const EnvironmentRow = ({ row, moveListItem }: IEnvironmentRowProps) => {
+    const dragItemRef = useDragItem(row.index, moveListItem);
+    const search = useSearchHighlightContext();
+    const draggable = !search;
```

Comment:
searchActive? I want to say this describes if a search is currently being used but it's not quite clear from the variable name?